lms/djangoapps/learner_home/mock_views.py

```diff
@@ -0,0 +1,2238 @@
+"""
+Mock implementation of the Learner Home.
+Returns statically authored JSON data
+"""
+# pylint: disable=line-too-long
+
+from django.contrib.auth.decorators import login_required
+from django.views.decorators.http import require_GET
+from common.djangoapps.util.json_request import JsonResponse
+
+
+# Edit me to change response data
+mock_data = {
```

Comment:
I feel like this should be in a json file.

Reply:
I second this. The previous incarnation, we were reading the data from `JSON`. Is there any reason to go this route instead? @nsprenkle

Reply:
Updated: Theoretically, I would want this to be decomposed into smaller units that could have active comments, generation, mutation, etc. but in this form, I agree. Moving to file